tests/e2e/cairo1/test_cairo1_building.py

```diff
@@ -9,14 +10,40 @@
 def test_cairo1_build(protostar: ProtostarFixture, copy_fixture: CopyFixture):
     copy_fixture("cairo1_build", "./cairo_project")
     os.chdir("./cairo_project")
-    protostar(["build-cairo1"])
+    result = protostar(["build-cairo1"])
+
+    assert 'Class hash for contract "main": 0x' in result
+    assert 'Compiled class hash for contract "main": 0x' in result
+
     compiled_sierra_path = Path("build/main.sierra.json")
     compiled_casm_path = Path("build/main.casm.json")
     assert compiled_sierra_path.exists()
     assert compiled_sierra_path.read_text()
     assert compiled_casm_path.exists()
     assert compiled_casm_path.read_text()
 
+    class_hash_path = Path("build/main.class.hash")
+    compiled_class_hash_path = Path("build/main.compiled.class.hash")
+    assert class_hash_path.exists()
+    assert class_hash_path.read_text().startswith("0x")
+    assert compiled_class_hash_path.exists()
+    assert compiled_class_hash_path.read_text().startswith("0x")
+
+
+def test_cairo1_build_json(protostar: ProtostarFixture, copy_fixture: CopyFixture):
+    copy_fixture("cairo1_build", "./cairo_project")
+    os.chdir("./cairo_project")
+    result_json = protostar(["build-cairo1", "--json"])
+
+    output_json_parsed = json.loads(
+        result_json.split("\n")[
+            3 if "This project expects older Protostar" in result_json else 0
+        ]
+    )
```

Comment:
Isn't this a bug? 
`This project expects older Protostar`
We shouldn't output this kind of info in plaintext if user expects ndjson on the output, right?

Reply:
But how the user will know he is using the wrong version of Protostar? Using the wrong version could result in unexpected behavior

Reply:
I think ndjson spec allows returning multiple jsons separated by newline character. Perhaps version warning should be returned as a separate json?

Reply:
It's out of scope probably even if it's true, but i thought it was worth noting. Perhaps we can create an issue for this kind of stuff? cc @abulenok

Reply:
warnings are redirected to stderr, ndjson should only be used in stdout. There is no need to do so.

Reply:
agree with @MaksymilianDemitraszek, for test I've added `ignore_stderr=True`